hilti/toolchain/src/compiler/cxx/unit.cc

```diff
@@ -273,6 +278,14 @@ void Unit::_generateCode(Formatter& f, bool prototypes_only) {
         }
     }
 
+    // Add the contents of the type information namespace. We know that there are only constants in there.
+    if ( ! prototypes_only ) {
+        for ( const auto& i : _constants ) {
+            if ( i.second.id.namespace_() == cxx::ID("__hlt::type_info::") )
+                f << i.second;
```

Comment:
Small prettification:

```suggestion
        for ( const auto& [id, constant] : _constants ) {
            if ( constant.id.namespace_() == cxx::ID("__hlt::type_info::") )
                f << constant;
```

Reply:
I'll leave as is for now as the surrounding code uses the same style.